index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Best Grill</title>
+     
+      <link rel="stylesheet" href="main.css">
+  </head>
+       <body>
+             <header>
+                    <div class="container">
+                        <div>
+                          <h2 class="logo"><a href="index.html"><img src="./components/resto.png" alt=""></a></h2>
```

Comment:
- [ ] In order to respect the given design that we need to follow, kindly, introduce to your project, for both Home and About pages, the top nav bar;
Given design:
![image](https://user-images.githubusercontent.com/83596135/163893620-b26bda52-bb1e-428a-8311-4351d217a4b3.png)